src/client/eppo-client.ts

```diff
@@ -115,7 +110,7 @@ export default class EppoClient implements IEppoClient {
         subjectAttributes,
         assignmentHooks,
         ValueType.StringType,
-      )?.stringValue ?? null
+      ).stringValue ?? null
```

Comment:
~why did we remove the optional chaining here and the other places? Can't `getAssignmentVariation()` return `null` in some classes? I think we should add it back lest we get "Cannot read property 'stringValue' of undefined" type errors.~

Ok I see the change and comment later down how we will always return an `EppoValue` and use the null flavor of that in cases we'd have null 👌